observability/src/test/java/io/grpc/observability/LoggingChannelProviderTest.java

```diff
@@ -126,6 +132,7 @@ public void newChannelBuilder_interceptorCalled() {
   }
 
   private static class NoopInterceptor implements ClientInterceptor {
+
```

Comment:
Can you get rid of this unnecessary change in this PR?

Reply:
Done